rita/src/rita_common/network_monitor/mod.rs

```diff
@@ -180,6 +184,63 @@ impl Default for NetworkMonitor {
     }
 }
 
+#[derive(Default, Serialize, Copy, Clone)]
+pub struct LatencyStats {
+    avg: Option<f32>,
+    std_dev: Option<f32>,
+}
+
+#[derive(Default, Serialize, Copy, Clone)]
+pub struct PacketLossStats {
+    avg: Option<f32>,
+    five_min_avg: f32,
+}
+
+pub struct GetStats {
```

Comment:
why exactly are we asking from the neighbours endpoint end to get a specific list of stats? If we just send what we have then the list should be the same and we don't risk crashing with .unwrap() if it's not.